script/GameObjects/Jobs/Mine.lua

```diff
@@ -114,17 +114,17 @@ end
 function Mine:onUpdateActive (e, dt)
   if not Config.game.gamePaused then
     Profiler.Begin('Actions.Mine.onUpdateActive')
-    if not e.jobState then e.jobState = 0 end
+    if not e.jobState then e.jobState = Config.Enums.JobStatesMine.None end
     e.jobState = e.jobState + 1
 
-    if e.jobState == 1 then
+    if e.jobState == Config.Enums.JobStatesMine.MovingToAsteroid then
```

Comment:
Why are Enums part of the Config Table? Doesn't make sense to me

Reply:
Should be just a separate global Enums table instead

Reply:
They are essentially static values, which also describes many/most of the values in App.lua. The game already references many things from the various Config tables, from rendering to debug to audio to ui to system generation to gameplay variables. So putting game-controlling enums in there as well didn't seem odd to me at all.

By "separate" do you mean a separate file, like a new "Enums.lua" in ./script/Config/ ? It shouldn't be too hard to do if there's some runtime or maintenance value in splitting enums off from the other game management values in ./script/Config/App.lua .

Reply:
1. A Config is a config to me 😅 just feels weird to put Enums in a config.

2. There is two ways you can do Enums. 
- Globally: Create one large Enum file which contains all Enums of LT e.g. global Enum file in Config/
- or define an Enum file for every category - create Folder Enums/ - create e.g. Systems.lua in there and define all Enums related with Systems in there. Do the same for all other Categories.

Reply:
I'd prefer the second option

Reply:
"define an Enum file for every category - create Folder Enums/ - create e.g. Systems.lua in there and define all Enums related with Systems in there. Do the same for all other Categories."

Done.